src/components/ZCLToolbar.vue

```diff
@@ -245,6 +243,12 @@ export default {
   name: 'ZCLToolbar',
   mixins: [CommonMixin],
   computed: {
+    query() {
+      return this.$store.state.zap.query // Access the query string from Vuex
+    },
+    showSaveButton() {
+      return !!this.query?.stsApplication // Use optional chaining and double negation for clarity
```

Comment:
What is stsApplication?

Reply:
the passed from studio to indicate which version it is..from the long discussions with Balint and @tecimovic